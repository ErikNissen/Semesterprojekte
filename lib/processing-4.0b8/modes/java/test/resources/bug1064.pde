version https://git-lfs.github.com/spec/v1
oid sha256:4bb96fe5d5a037a907b23cedf150bcefa3aec0ccb9caed2bc138c7dad09f0df3
size 12
