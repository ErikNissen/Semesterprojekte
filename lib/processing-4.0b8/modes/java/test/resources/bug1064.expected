version https://git-lfs.github.com/spec/v1
oid sha256:bf90b2a12b60ba558c931a167ea95cdd9a01e538dba1ac918be9750b9c72e6d2
size 706
